### PR TITLE
Issue #139 Add memory and cpu options to config

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: cmd/crc/cmd/start.go [hunk 1]
```diff
@@ -36,7 +37,7 @@ var (
 )
 
 func runStart(arguments []string) {
-	if err := constants.ValidateDriver(crcConfig.GetString(config.VMDriver.Name)); err != nil {
+	if err := validateStartFlags(); err != nil {
```
Comment (anjannath) [inline cmd/crc/cmd/start.go]: :+1:

File: cmd/crc/cmd/config/config.go [hunk 2]
```diff
@@ -22,6 +23,8 @@ var (
 	// Start command settings in config
 	VMDriver = createSetting("vm-driver", nil, []validationFnType{validations.ValidateDriver})
 	Bundle   = createSetting("bundle", nil, nil)
+	CPUs     = createSetting("cpus", constants.DefaultCPUs, []validationFnType{validations.ValidateCPUs})
+	Memory   = createSetting("memory", nil, []validationFnType{validations.ValidateMemory})
```
Comment (gbraad) [inline cmd/crc/cmd/config/config.go]: We have a default for memory...

we currently take the default from the bundle...  but that would be the same for the CPUs, so why different here?

File: cmd/crc/cmd/start.go [hunk 3]
```diff
@@ -46,8 +47,8 @@ func runStart(arguments []string) {
 		Name:       constants.DefaultName,
 		BundlePath: crcConfig.GetString(config.Bundle.Name),
 		VMDriver:   crcConfig.GetString(config.VMDriver.Name),
-		Memory:     constants.DefaultMemory,
```
Comment (gbraad) [inline cmd/crc/cmd/start.go]: DefaultMemory has been defined, but not used anymore?

File: pkg/crc/validation/validation.go [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+func ValidateCPUs(value int) error {
+	if value < constants.DefaultCPUs {
+		return fmt.Errorf("CPUs required >=%d", constants.DefaultCPUs)
+	}
+	return nil
+}
+
+// ValidateMemory is check if provided Memory count is valid
+func ValidateMemory(value int) error {
+	if value < constants.DefaultMemory {
```
Comment (gbraad) [inline pkg/crc/validation/validation.go]: Used in the validation, but not as default...

File: cmd/crc/cmd/config/config.go [hunk 5]
```diff
@@ -20,8 +21,10 @@ var SettingsList = make(map[string]*setting)
 
 var (
 	// Start command settings in config
-	VMDriver = createSetting("vm-driver", nil, []validationFnType{validations.ValidateDriver})
-	Bundle   = createSetting("bundle", nil, nil)
+	VMDriver = createSetting("vm-driver", constants.DefaultVMDriver, []validationFnType{validations.ValidateDriver})
+	Bundle   = createSetting("bundle", nil, []validationFnType{validations.ValidateBundle})
+	CPUs     = createSetting("cpus", constants.DefaultCPUs, []validationFnType{validations.ValidateCPUs})
+	Memory   = createSetting("memory", constants.DefaultMemory, []validationFnType{validations.ValidateMemory})
```
Comment (gbraad) [inline cmd/crc/cmd/config/config.go]: :+1:

**Pull request merged**